docs/src/pages/system/box/BoxSx.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Box } from '@material-ui/system';
+import { ThemeContext } from '@material-ui/styled-engine';
+
+export default function BoxSx() {
+  return (
+    <ThemeContext.Provider
```

Comment:
Should we just use `import { ThemeProvider } from '@emotion/react'`? I want to emphasize that you don't need to depend on @material-ui/core.

Reply:
I would personally keep the current content in the demo, to limit the dependency on emotion as much as we can.